.eslintrc.json

```diff
@@ -40,28 +50,57 @@
       }
     ],
     "keyword-spacing": "error",
+    "linebreak-style": "error",
+    "max-statements-per-line": "error",
```

Comment:
I was hoping to forbid future additions of multiple statements like `pos.x = x; pos.y = y;`